src/test/java/uk/gov/dhsc/htbhf/page/BasePage.java

```diff
@@ -33,16 +37,16 @@ public void open() {
         webDriver.get(fullUrl);
     }
 
-    public WebElement findByClassName(String className) {
-        return webDriver.findElement(By.className(className));
+    public void waitForPageToLoad() {
+        wait.until(ExpectedConditions.titleIs(getPageTitle()));
     }
 
-    public WebElement findByCss(String cssSelector) {
-        return webDriver.findElement(By.cssSelector(cssSelector));
+    public List<WebElement> getRadioButtons() {
+        return webDriver.findElements(By.className("govuk-radios__item"));
```

Comment:
Do we want these class names to be constants somewhere? Maybe not.

Reply:
I was in two mind about this at they'll probably only be used here, but I'll add them as constants.